sympy/solvers/ode/tests/test_systems.py

```diff
@@ -1922,3 +2018,77 @@ def test_dsolve():
     eqs = [f(x, y).diff(x)+g(x).diff(x), g(x).diff(x)]
     with raises(ValueError):
         dsolve(eqs)
+
+
+def _higher_order_slow1():
+    x, y = symbols("x y", cls=Function)
+    t = symbols("t")
+
+    eq = (Eq(diff(x(t),t,t), (log(t)+t**2)*diff(x(t),t)+(log(t)+t**2)*3*diff(y(t),t)), Eq(diff(y(t),t,t), \
+    (log(t)+t**2)*2*diff(x(t),t)+(log(t)+t**2)*9*diff(y(t),t)))
+    _x1 = log(t)**2
+    _x2 = sqrt(22)
+    _x3 = sqrt(9*_x1 + t**4 + 6*t**2*log(t) - 6*t**2 - 18*log(t) + 9)
```

Comment:
We need to find a better way of handling examples like this. Rather than using the simpsol function to create the code for enormous complicated test solutions we should use it to study the structure of the solutions and how to make them simpler. If they can't be made simpler then we need tests that give simpler answers because these are just unmanageable.

If someone changed something somewhere and it made a minus sign or something appear somewhere in this solution then they would need to regenerate this test and they would find it very difficult to do that. This needs to be written in a way that is understandable and updateable.

Reply:
So, should we remove these type of test cases which were made simpler using your simplification method?